### DISCUSSION TITLE
2024-08-02 weekly

### DISCUSSION THREAD

**Discussion by chickenlj** [apache/dubbo-go#2724] [General]
## 会议纪要
本轮任务与维护工作，由社区Committer 唐启文负责推进

### 任务一：配置统一，解决配置不生效、位置分散、启动流程不清晰等问题。唐启文、王一博、徐战潮一起跟进
1. config与global.config配置整合，当初global.config从设计上是取代config，目前还未全部完成
2. config掏空，保留老用户兼容性
3. config包中的启动动作迁移，移到server/client等位置
4. 开放一些协议层配置

### 任务二：包依赖关系。张文锋
一些影响比较大的相关包如下：
● common/extension
● config/global
● server/client
● protocol
● registry/metadata

### 任务四：triple协议。徐战潮、如漫
1. triple协议的链接与可用性管理，参考grpc-go等keepalive机制。徐战潮、如漫
2. triple与dubbo3/grpc之间的关系，徐战潮

### 其他任务：
● 优雅停机
● 服务发现
● xds，王一博
● java转go的proto工具

**Comment by AlexStocks** [comment]
@chickenlj dubbogo 内存泄漏问题，https://github.com/No-SilverBullet 近期会提交一个 PR 上来。
